scripts/ci/ci-run-stateful-tests-cluster.sh

```diff
@@ -30,14 +30,14 @@ else
     exit 1
 fi
 
-./target/release/bendctl --databend_dir ./.databend --group local query ./tests/suites/0_stateful/ddl/create_table.sql
+./target/debug/bendctl --databend_dir ./.databend --group local query ./tests/suites/0_stateful/ddl/create_table.sql
```

Comment:
Why change to `debug`?

Reply:
to reduce ci compile time，similar to stateless tests

Reply:
How long does it take to run the stateful test in debug mode? Is the total time (compile + run) actually reduced?

Reply:
The debug mode in loading is slower than release, bendctl release compile time is more but the load data is less, I prefer to use release to do the stateful test.

Reply:
I shall use release mode once deployed runner into cloud service